ui/component/publishForm/view.jsx

```diff
@@ -180,12 +253,39 @@ function PublishForm(props: Props) {
     }
   }, [modal]);
 
-  const isLivestream = mode === PUBLISH_MODES.LIVESTREAM;
+  // move this to lbryinc OR to a file under ui, and/or provide a standardized livestreaming config.
+  function checkLivestreams(channelId, signature, timestamp) {
+    // $FlowFixMe Bitwave's API can handle garbage
+    const fakeId = '2bfe6cdb24a21bdc1b76fb7c416edd50e9e85945'; // remove this when done testing
+    setCheckingLivestreams(true);
+    fetch(`https://api.bitwave.tv/v1/replays/odysee/${channelId}?signature=${signature}&signing_ts=${timestamp}`) // claimChannelId
```

Comment:
this should use `BITWAVE_API` from `constants/livestream`

Reply:
also remove the comment

Reply:
ok, did this
```
export const BITWAVE_LIVE_API = 'https://api.bitwave.tv/v1/odysee/live';
export const BITWAVE_REPLAY_API = 'https://api.bitwave.tv/v1/replays/odysee';
```